lusSTR/filter.py

```diff
@@ -86,8 +95,8 @@ def process_strs(dict_loc, datatype):
             columns=[
                 *data_order.columns.tolist(),
                 "allele_type",
-                "stuttering_allele1",
-                "stuttering_allele2",
+                "parent_allele1",
+                "parent_allele2",
```

Comment:
I changed this column name from ```stuttering_alleleX``` to ```parent_alleleX```. I saw that STRmix uses "parent allele" and I liked that much better.